jbmc/src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -1049,6 +1070,15 @@ bool java_bytecode_languaget::convert_single_method(
   symbol_table_baset &symbol_table,
   optionalt<ci_lazy_methods_neededt> needed_lazy_methods)
 {
+  // Method is not in context, hence we don't convert it,
+  // unless it is a clinit_wrapper.
+  if(
+    !is_clinit_wrapper_function(function_id) && method_in_context &&
```

Comment:
I don't want this to merged as is. Currently, clinit_wrappers append a `::clinit_wrapper` suffix directly to the class name, which makes the structure of the name different from other methods. 
We have to fix a naming convention for synthetic methods that doesn't produce unintended side-effects. What about 'class`.<clinit_wrapper>`'?

Reply:
Fine with me